pyuvdata/uvdata/tests/test_uvdata.py

```diff
@@ -1729,14 +1729,10 @@ def test_select_bls(casa_uvfits):
 
     with pytest.raises(ValueError) as cm:
         uv_object.select(bls=(5, 1))
-    assert str(cm.value).startswith(
-        "Antenna number 5 is not present in the " "ant_1_array or ant_2_array"
-    )
+    assert str(cm.value).startswith("Antenna pair (5, 1) does not have any data")
```

Comment:
I actually prefer to use the `match` parameter to pytest raises like this:
```
with pytest.raises(ValueError, match="Antenna pair (5, 1) does not have any data"):
         uv_object.select(bls=(5, 1))
```

I've been slowly converting all of the tests that use this form to use that one, I'd be happy for help!

Reply:
Converted all of them in this test :-)